modules/visual_script/editor/visual_script_editor.cpp

```diff
@@ -2727,7 +2727,7 @@ void VisualScriptEditor::_center_on_node(int p_id) {
 
 	if (gn) {
 		gn->set_selected(true);
-		Vector2 new_scroll = gn->get_position_offset() - graph->get_size() * 0.5 + gn->get_size() * 0.5;
+		Vector2 new_scroll = Vector2(gn->get_position_offset().x * graph->get_zoom() - graph->get_size().x / 2 + gn->get_size().x * 0.5, gn->get_position_offset().y * graph->get_zoom() - graph->get_size().y / 2 + gn->get_size().y * 0.5);
```

Comment:
Why not use the Vector2 operators like the previous code? This would be much more readable.
```suggestion
		Vector2 new_scroll = gn->get_position_offset() * graph->get_zoom() - graph->get_size() * 0.5 + gn->get_size() * 0.5;
```